conanfile.py

```diff
@@ -160,8 +162,9 @@ def configure(self):
             for opt, val in self.options.values.items():
                 if opt.startswith('build_'):
                     setattr(self.options, opt, True)
-        if not self.options.celix_cxx:
+        if not self.options.celix_cxx14:
             self.options.build_cxx_remote_service_admin = False
+        if not self.options.celix_cxx17:
```

Comment:
```suggestion
        if not self.options.celix_cxx17:
                self.options.build_cxx_remote_service_admin = False
```

According to the corresponsing CMakeLists.txt, cxx_remote_service_admin is C++17.